crates/solidity/src/solc/standard_json/input/settings/selection/file/flag.rs

```diff
@@ -40,6 +40,12 @@ pub enum Flag {
     /// The assembly code
     #[serde(rename = "evm.assembly")]
     Assembly,
+    /// The Ir
+    #[serde(rename = "ir")]
+    Ir,
+    // Catch all unsupported flags
+    #[serde(other)]
+    Unknown,
```

Comment:
I don't like this. I'd prefer to be explicit about the output selection. If there is a bug we can fix it and if `solc` adds new one we should catch it and implement those properly.

Reply:
removed it

Reply:
my bad, forgot that serde already generates tolerable error messages for simple enums.